### PR TITLE
Update Changelog and version

### PR DESCRIPTION
**Pull request by fabrixxm**

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
+	Delete attachments on item deletion, delete videos from video tab (issue #1574) (fabrixxm)
+	Improvements with reshared Diaspora items (annando)
+	Improvements in OStatus communications: (annando)
+		improve duplicate handling
+		publish comments to post to all PuSH subscribers
+		use correct contact when automatically add @-replies
+		add attachment links as enclosures
+		send salmon notifications to every mentioned person
+		better thread completition
+		support to bookmarks
```
Comment (annando) [inline CHANGELOG]: Support *for*

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,55 @@
...
+		support to events and questions
```
Comment (annando) [inline CHANGELOG]: Support *for*

File: CHANGELOG [hunk 3]
```diff
@@ -1,3 +1,55 @@
...
+		link to items using GUID
+	Fix warining in mod/photo (issue #1638) (rabuzarus)
```
Comment (annando) [inline CHANGELOG]: warning

File: CHANGELOG [hunk 4]
```diff
@@ -1,3 +1,55 @@
...
+	New option to block public access to local directory and poco
+	Fix parsing bbcode [url] tag with fragment identifier (issue #1514) (fabrixxm)
+	Fix HTML for oembeds (issue #1612) (fabrixxm)
+	Add fake filelds to API response for better Twitter API compatibility (annando)
```
Comment (annando) [inline CHANGELOG]: fake *fields*

File: CHANGELOG [hunk 5]
```diff
@@ -1,3 +1,55 @@
...
+	Fix double html encodig in site administration page for sitename and register text (issue #1628) (annando)
+	Fix remote subscription from GNU Social (annando)
+	Fix "{0}" in notifications (issue #1642) (annando)
+	Fix desktop notification (fabrixxm)
+	Fix rewrite test in install wizard with self-signed certificate (annando)
+	Better support for non standard installations of GNU Social (annando)
+	Fix emoticons alt text (tobias)
+	Improve threaded display in Vier theme (annando)
+	Use field templates in photo edit form (fabrixxm)
+	Alllow deletion to any user but yourself (issue #1625) (fabrixxm)
```
Comment (annando) [inline CHANGELOG]: *of* any user

**Comment by fabrixxm** [comment]
ehm...

File: CHANGELOG [hunk 6]
```diff
@@ -1,3 +1,55 @@
...
+	Install wizard load htconfig template from template/ folder, remove localized htconfig templates (fabrixxm)
+	Add contact detail to non-js contact drop confirm dialog (issue #1629) (fabrixxm)
+	Return geo coord in API (annando)
+	Improve events reminder: use title, show in colorbox, link using event ID (rabuzarus)
+	Fix spelling in accepted connection notification email (strk)
+	Show image size warning in a human readable format (rabuzarus)
+	Move ACL window in template (rabuzarus)
+	New option "-s" in util/run_xgettext.sh (fabrixxm)
+	Support but ignore delete message from Quitter (annando)
+	Remove google maps from core (issue #1705) (annando)
```
Comment (tobiasd) [inline CHANGELOG]: maybe more precise "moved to addon"? and mention choice OSM/GM

File: CHANGELOG [hunk 7]
```diff
@@ -1,3 +1,55 @@
...
+	Alllow deletion of any user but yourself (issue #1625) (fabrixxm)
+	Install wizard load htconfig template from template/ folder, remove localized htconfig templates (fabrixxm)
+	Add contact detail to non-js contact drop confirm dialog (issue #1629) (fabrixxm)
+	Return geo coord in API (annando)
+	Improve events reminder: use title, show in colorbox, link using event ID (rabuzarus)
+	Fix spelling in accepted connection notification email (strk)
+	Show image size warning in a human readable format (rabuzarus)
+	Move ACL window in template (rabuzarus)
+	New option "-s" in util/run_xgettext.sh (fabrixxm)
+	Support but ignore delete message from Quitter (annando)
```
Comment (tobiasd) [inline CHANGELOG]: I don't get this one ;-)

**Pull request merged**